pkg/agent/types/multicast.go

```diff
@@ -36,8 +36,7 @@ var (
 )
 
 type MulticastValidator interface {
-	// Validate checks whether IGMP report from Pod(podNamespace/podName) to groupAddress should be dropped,
+	// GetMcastNPRuleInfo checks whether IGMP report from Pod(podNamespace/podName) to groupAddress should be dropped,
```

Comment:
Pod -> a Pod

Reply:
It looks up any rule applied, not just "dropped"?

Reply:
done